### PR TITLE
Various translatability additions

### PR DESCRIPTION
**Pull request by Ckrapock**

### CODE REVIEW

File: lang/fr.json [hunk 1]
```diff
@@ -885,13 +885,13 @@
     "Åland Islands": "Les îles d'Åland",
 	"Message restored" : "Message restauré",
 	"Permanent ban" : "Ban permanent",
-	" has been banned.": " a été banni",
+	" has been banned.": " a été banni.",
```
Comment (mchev) [inline lang/fr.json]: Globalement ne pas mettre les ponctuations et espaces dans les traductions mais plutôt dans les vues.

Comment (Ckrapock) [reply lang/fr.json]: Yes tu as raison! Alors... j'ai commencé un petit ménage sur certains (WIP), mais j'ai un doute sur pas mal d'autres.
Peux-tu me donner la syntaxe du vue pour cet exemple, si on voulait le transformer en `"has been banned":"a été banni",`

Comment (mchev) [reply lang/fr.json]: Celui la n'est utilisé que dans le fichier `App\Http\Controllers\UserBanController.php`

Pour remplacer l'espace :
**Avant**
`return redirect()->back()->with('success', $user->name.__(' has been banned.'));`
**Après** 
`return redirect()->back()->with('success', $user->name. ' ' . __('has been banned.'));`

File: resources/js/Components/Notifications/NewSuggestion.vue [hunk 2]
```diff
@@ -15,7 +15,7 @@ const emit = defineEmits(['markedAsdone'])
       {{ notification.data.message }}
     </div>
     <span class="flex text-xs"
-      > {{ __('Sent by ') }}{{ notification.data.user.name }} {{ __('on ') }}<span class="ml-1 font-bold">{{ notification.data.room.name }}</span></span
+      > {{ __('Sent by ') }}{{ notification.data.user.name }} {{ __('on ') }} <span class="ml-1 font-bold">{{ notification.data.room.name }}</span></span
```
Comment (mchev) [inline resources/js/Components/Notifications/NewSuggestion.vue]: Par exemple ici.

`{{ __('Sent by ') }}{{ notification.data.user.name }} {{ __('on ') }} <span class="ml-1 font-bold">{{ notification.data.room.name }}`

devient

`{{ __('Sent by') }}&nbsp;{{ notification.data.user.name }}&nbsp;{{ __('on') }}&nbsp;<span class="font-bold">{{ notification.data.room.name }}`

`&nbsp;` : Non breaking space HTML Character Entity

File: lang/fr.json (hunk 1 above)
Comment (Ckrapock) [reply lang/fr.json]: Merci ! 
et le point à la fin, comment le remplacer ? comme dessous ?
return redirect()->back()->with('success', $user->name. ' ' . __('has been banned'). '.' );

File: resources/js/Components/Notifications/NewSuggestion.vue (hunk 2 above)
Comment (Ckrapock) [reply resources/js/Components/Notifications/NewSuggestion.vue]: dac c'est noté merci!

**Comment by Ckrapock** [comment]
@mchev je pense avoir fini ce "round" de traductions. Pour la ponctuation et les points à la fin des libellés, je dirais que ce serait pas mal de les garder dans le fichier de traduction sans le rapatrier dans les vues/php, pour la lisibilité, mais je fais comme tu me diras ! Si on rapatrie les points dans les vue/php, peux-tu me confimer cette syntaxe suite à ma question ici : https://github.com/mchev/blinest/pull/74#discussion_r1148550053

**Review by mchev — COMMENTED** [review]
The last item of an array doesn't require a comma. (Same for PHP and JS)

**Comment by mchev** [comment]
`return Redirect::back()->with('success', __('Room updated.'));`
Should be :
`return Redirect::back()->with('success', __('Room updated'));`

The goal is to simplify every translated string to the minimum. You also can remove a string if not stricly necessary.

For punctuation, we remove all the periods '.' wherever they are. On the other hand, the exlamation marks are left in because, for example, in Spanish (if one day it happens) the punctuation must be translated as well.

Ideally we could also clean the fr.json file because I'm sure there are strings that are not used anywhere on the site.

I'll leave the PR open as a basis for definitively fixing the translations on the site.

**Comment by Ckrapock** [comment]
Got it @mchev we're not far.

- The last item of an array doesn't require a comma.
**OK I'll take it !** Will remove in both messages.php from FR and EN.  I added it because it was made like this in the French message.php file you recently updated.
- For punctuation, we remove all the periods '.' wherever they are.
**OK I'll take it !** Will remove all "." in strings or vue/php files AND fr.json. 
Regarding the "?" or "!" or ";" yes fully agree we need to keep them. Even between FR and EN there is a difference of syntax convention. FR adds a space, whereas EN doesn't have a space for question marks and other similar signs.
- Unused strings cleanup
**OK I'll take it !**.... but ... it will take me a little more time and i'll have to check one by one, unless you have a way to have all unused ones. Well maybe not one by one exactly as I doubt Saint Kitts & Nevis is used anywhere, but still i'll need to be careful !

**What I suggest :** I'll do the 2 first points quickly today, then I'll start the point 3, but you can merge when i don't have point 3 completed yet, in case you have other developments to merge (if possible avoiding me to do a perilous synch of branches on the way :) ).

**Comment by Ckrapock** [comment]
@mchev points 1 and 2 are done. Point 3 on unused strings cleanup is started, by removing all countries.

From now on you can merge any time (if all the rest ok for you)
While you don't merge i'll do regular unused string cleanup as part of this PR.
If you merge i'll start another PR for new cleanup iterations.

**Pull request merged**